### PR TITLE
New deprecated warnings included with Kube 1.19

### PR DESCRIPTION
**Pull request by codyhoag**

### CODE REVIEW

**Comment by codyhoag** [comment]
@soltysh PTAL? Thanks!

Preview: http://file.rdu.redhat.com/~choag/090420/rn-4.6-kub-warnings/release_notes/ocp-4-6-release-notes.html#ocp-4-6-notable-technical-changes

**Review by soltysh — APPROVED** [review]
/lgtm

File: release_notes/ocp-4-6-release-notes.adoc [hunk 1]
```diff
@@ -103,11 +103,29 @@ By default, the following network connections are checked:
...
 [id="ocp-4-6-ovn-k8s-default-cni-np-uses-ovs-on-cluster-nodes"]
 ==== OVN-Kubernetes default CNI network provider now uses OVS installed on cluster nodes
 
 The OVN-Kubernetes default Container Network Interface (CNI) network provider now uses the Open vSwitch (OVS) version installed on the cluster nodes. Previously, OVS ran in a container on each node, managed by a DaemonSet. Using the host OVS eliminates any possible downtime from upgrading the containerized version of OVS.
 
+[discrete]
+[id="ocp-4-6-warnings-when-using-deprecated-apis"]
+==== Warnings when using deprecated APIs
+
+Warnings are now visible in `client-go` and `oc/kubectl` on every invocation
```
Comment (bergerhoffer) [inline release_notes/ocp-4-6-release-notes.adoc:116]: I think you can remove "kubectl" here - we usually just show "oc" throughout our docs.

File: release_notes/ocp-4-6-release-notes.adoc [hunk 2]
```diff
@@ -22,7 +22,7 @@ compliance, and governance requirements.
 
 Red Hat {product-title}
 (link:https://access.redhat.com/errata/RHBA-2020:1234[RHBA-2020:1234]) is now
-available. This release uses link:https://v1-18.docs.kubernetes.io/docs/setup/release/notes/[Kubernetes 1.18] with CRI-O runtime. New features, changes, and known issues that pertain to
+available. This release uses link:https://kubernetes.io/docs/setup/release/notes/#v1-19-0[Kubernetes 1.19] with CRI-O runtime. New features, changes, and known issues that pertain to
```
Comment (bergerhoffer) [inline release_notes/ocp-4-6-release-notes.adoc:25]: So, I had a PR (#24322) prepped to make this link update, but was waiting on the URL to work for 1.19. However once 1.19 came out, it doesn't seem like they posted one like this.

My concerns with using this current URL (without the preceding v1-19) is that these will probably break once 1.20 comes out. Not sure if I have an alternative solution to suggest, but bringing up for thoughts, wdyt?

Same for the link on line 127.

File: release_notes/ocp-4-6-release-notes.adoc [hunk 3]
```diff
@@ -103,11 +103,29 @@ By default, the following network connections are checked:
...
 
 The OVN-Kubernetes default Container Network Interface (CNI) network provider now uses the Open vSwitch (OVS) version installed on the cluster nodes. Previously, OVS ran in a container on each node, managed by a DaemonSet. Using the host OVS eliminates any possible downtime from upgrading the containerized version of OVS.
 
+[discrete]
+[id="ocp-4-6-warnings-when-using-deprecated-apis"]
+==== Warnings when using deprecated APIs
+
+Warnings are now visible in `client-go` and `oc/kubectl` on every invocation
+against a deprecated API. Calling a deprecated API returns a warning message
+containing the target removal release and replacement API, if applicable.
```
Comment (bergerhoffer) [inline release_notes/ocp-4-6-release-notes.adoc:118]: I'd specify that the target removal release is a Kubernetes release (not OCP).

File: release_notes/ocp-4-6-release-notes.adoc (hunk 2 above)
Comment (codyhoag) [reply release_notes/ocp-4-6-release-notes.adoc:25]: I see. I'll revert that link on line 25 back to 1.18 for now and rely on your PR to update it when the Kube 1.19 URL is available. I'll just call out Kube 1.19 without the link on line 127 :+1:

**Pull request merged**